static/app/views/settings/project/projectTeams.spec.tsx

```diff
@@ -257,13 +234,10 @@ describe('ProjectTeams', function () {
       statusCode: 200,
     });
 
-    render(
-      <ProjectTeams
-        {...RouteComponentPropsFixture()}
-        params={{projectId: project.slug}}
-        organization={org}
-        project={project}
-      />
+    render(<ProjectTeams organization={org} project={project} />);
+
+    await waitFor(() =>
+      expect(screen.getByText('Project Teams for project-slug')).toBeInTheDocument()
```

Comment:
why no findBy
```suggestion
      expect(await screen.findByText('Project Teams for project-slug')).toBeInTheDocument()
```

Reply:
You're right - think this was a copilot autocomplete